Source/WPEProcess/Process.cpp

```diff
@@ -649,7 +649,22 @@ int main(int argc, char** argv)
             if (options.Group != nullptr) {
                 Core::ProcessCurrent().Group(string(options.Group));
             }
+
+            Core::Directory dataPath(options.DataPath.c_str());
+            dataPath.Permission((S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP));
```

Comment:
See previous remark, remove the changes on the dataPath

Reply:
removed